test/index.js

```diff
@@ -79,6 +79,29 @@ test('roundtrip', (t) => {
   setTimeout(() => outTransport.destroy(), 100);
 });
 
+test('error on createStream if destroyed', (t) => {
+  const stream = new ObjMultiplex();
+  stream.destroy();
+  try {
+    stream.createStream('controller');
+  } catch (e) {
+    t.assert(e.message.includes('already destroyed'), true);
+    t.end();
+  }
+});
+
+test('error on createStream if ended', (t) => {
+  const stream = new ObjMultiplex();
+  stream.end();
+  try {
+    stream.createStream('controller');
+    stream.write({ foo: 'bar' });
```

Comment:
It looks like this was left in during testing - I tested that it works without this line. This might wrongly imply that the error is thrown during the write, which is exactly what we want to prevent here.

```suggestion
```